packages/govuk-frontend/src/govuk/components/error-summary/error-summary.mjs

```diff
@@ -22,14 +23,11 @@ export class ErrorSummary {
    * @param {ErrorSummaryConfig} [config] - Error summary config
    */
   constructor ($module, config) {
-    // Some consuming code may not be passing a module,
-    // for example if they initialise the component
-    // on their own by directly passing the result
-    // of `document.querySelector`.
-    // To avoid breaking further JavaScript initialisation
```

Comment:
I was quite fond of this message, all on its own in **ErrorSummary**

Thoughts on adding a little disclaimer to every component?

1. Error throws
2. Error console shows stack trace
3. Stack trace uses original source via source maps
4. Developer clicks to these lines and reads helpful message

Reply:
I think most of that info should be summarised in the error message when implementing `https://github.com/alphagov/govuk-frontend/issues/4035`.

Regarding adding things to each components, I think this would rather be the role of a base Component class. I think with the error throwing for GOV.UK Frontend support and the type check of `$module` there's enough ground for creating it already (only to add new features to it, not refactor existing code into it) and centralise these checks and possible comments.